webpack.config.js

```diff
@@ -37,6 +38,9 @@ const webpackConfig = {
 		chunkFilename: '[name].[chunkhash].js', // ditto
 		devtoolModuleFilenameTemplate: 'app:///[resource-path]'
 	},
+	postcss() {
```

Comment:
It'll maybe useful to take a look at #665-gh-delphin where a similar issue is addressed